flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultSubpartition.java

```diff
@@ -158,11 +158,18 @@ public boolean add(BufferConsumer bufferConsumer) throws IOException {
 		private final Buffer buffer;
 		private final boolean isDataAvailable;
 		private final int buffersInBacklog;
+		private final int unannouncedBacklog;
```

Comment:
`buffersInBacklog` and `unannouncedBacklog` should be retained only one finally.